<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>569</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>569</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>760</width>
    <height>569</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(246, 222, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Шифрование</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Расшифровка</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>321</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>321</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>380</y>
     <width>231</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 221, 235);</string>
   </property>
   <property name="text">
    <string>Зашифровать в файл</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>380</y>
     <width>221</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 221, 235);</string>
   </property>
   <property name="text">
    <string>Расшифровать файл</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>450</y>
     <width>201</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 221, 235);</string>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
